MetricsUploader/include/MetricsUploader/MetricsUploader.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+#define ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+
+#ifdef _WIN32
+#include <windows.h>
+#endif  // WIN32
+
+#include "orbit_log_event.pb.h"
+
+namespace orbit_metrics_uploader {
+
+/**
+ * This class is used for sending log events from Orbit. It works only on Windows and only if
+ * metrics_uploader_client.dll is available. The types of logs that could be send are defined in
+ * orbit_client_protos::OrbitLogEvent::LogEventType.
+ */
```

Comment:
The text also needs to describe the pre-conditions. In this case: The class can only be instantiated once. But like I said in other comments, ideally we make that clear on the API level and don't rely on comments or CHECKS.

Reply:
I've changed the description, ptal